css/style.css

```diff
@@ -0,0 +1,970 @@
+body {
+  font-family: "Roboto Condensed", sans-serif;
+  max-width: 1300px;
+}
+
+.news-logo {
+  width: 400px;
+  margin-bottom: -20px;
+}
+
+nav {
+  background-color: #f72210;
+}
+
+a {
+  text-decoration: none;
+}
+
+.nav-1 {
+  height: 90px;
+}
+
+svg {
+  fill: white;
+}
+
+nav > section > div > a {
+  margin-right: 20px;
+  font-weight: bolder;
+}
+
+.nav-0 > a {
+  padding-bottom: 8px;
+}
+
+.nav2 {
+  margin-bottom: -5px;
+}
+
+.newsletter > a {
+  color: white;
+  background-color: #f72210;
+  font-weight: bold;
+  border: none;
+  border-radius: 2px;
+  line-height: 31px;
+}
+
+.last > a {
+  border-right: none !important;
+}
+
+.text > a,
+.text1 > div > a {
+  color: white;
+  background-color: #f72210;
+  font-weight: bold;
+  border: none;
+  border-radius: 2px;
+  line-height: 31px;
+  width: 120px;
+}
+
+.human {
+  font-size: 0;
+  background: url(https://web.archive.org/web/20210120125445im_/https://g.newsweek.com/www/images/user-w.svg) center / contain no-repeat;
+  width: 30px;
+  height: 30px;
+  padding-left: 0;
+  opacity: 0.75;
+  color: white;
+  margin-right: 8px;
+}
+
+.nav2 > ul > li > a {
+  font-size: 19px;
+  color: black;
+  font-weight: bold;
+  text-align: center;
+  flex-grow: 1;
+  border-right: 1px solid rgb(221, 221, 221);
+}
+
+.nav-3 {
+  border-top: 2px solid #f4f5f7;
+  border-bottom: 2px solid #f4f5f7;
+}
+
+.nav-3 > ul > li > a {
```

Comment:
- [ ] [OPTIONAL] Consider opting for the use of specific classes instead of long complex selectors like this one.
If you use selectors heavily based on tags that makes your code potentially very breakable since any change to the HTML will break the styling, in that way classes are much more flexible, sturdy, and usually the best choice when styling a project.